### PR TITLE
[TACHYON-988] Added LocalTachyonClusterResource to more tests

### PR DESCRIPTION
**Pull request by manugoyal**
There were other tests outside of integration-tests that could
use LocalTachyonClusterResource, so this makes use of it in those
tests. This requried moving the class to the minicluster tests
directory, and modifying the pom.xml of any module that needs it.

### CODE REVIEW

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by aaudiber** [comment]
Should we instead move these tests to the integration-tests project? If they need to fire up and interact with an actual local cluster, they aren't really unit tests.

**Comment by gpang** [comment]
I think it is strange that there are "unittests" that need the local cluster. Is there some limitation for just moving it to integration-tests?

**Comment by gpang** [comment]
Also, please resolve merge conflicts.

File: minicluster/pom.xml [hunk 1]
```diff
@@ -19,8 +19,30 @@
     <findbugs.path>${project.parent.basedir}/build/findbugs/</findbugs.path>
   </properties>
 
+  <build>
```
Comment (apc999) [inline minicluster/pom.xml]: NIT: to keep consistency with other pom files, could you move this `<build>` session after `<dependencies>` session?

File: minicluster/pom.xml [hunk 2]
```diff
@@ -19,8 +19,30 @@
...
+    <plugins>
+     <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-jar-plugin</artifactId>
+       <version>2.2</version>
```
Comment (apc999) [inline minicluster/pom.xml]: I think we don 't need line 27 as in parent pom file, `maven-jar-plugin` has been specified in `<pluginManagement>`  to use version 2.4

**Comment by apc999** [comment]
Thanks for the improvement, I left a few comments. Please also solve merge conflicts.

**Comment by manugoyal** [comment]
I went with the suggestion to move all the tests that used a LocalTachyonClusterResource into the integration-tests directory. That way, we don't have to modify too many pom.xmls.

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java [hunk 3]
```diff
@@ -60,6 +61,9 @@
  */
 public class TfsShellTest {
   private static final int SIZE_BYTES = Constants.MB * 10;
+  @Rule
+  public LocalTachyonClusterResource mLocalTachyonClusterResource =
+      new LocalTachyonClusterResource(SIZE_BYTES, 1000, Constants.GB);
   private LocalTachyonCluster mLocalTachyonCluster = null;
```
Comment (gpang) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: does this test need both 'LocalTachyonClusterResource' above, and 'LocalTachyonCluster' here?

File: shell/src/test/resources/log4j.properties [hunk 4]
```diff
@@ -1,17 +0,0 @@
-tachyon.root.logger=INFO, TEST_LOGGER
```
Comment (gpang) [inline shell/src/test/resources/log4j.properties]: why was this file removed?

**Comment by gpang** [comment]
Thanks for moving those tests to integration-tests! I left a few questions.

**Comment by aaudiber** [comment]
LGTM

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java (hunk 3 above)
Comment (manugoyal) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: Yeah I left it in there because a lot of methods call methods on the `mLocalTachyonCluster`, so rather than modify all of them, I just made it a local variable. Only the `LocalTachyonClusterResource` actually manages a cluster, though.

On a side note, I noticed there was some explicit cluster configuration that can be moved to the cluster resource constructor, so I'll change that.

File: shell/src/test/resources/log4j.properties (hunk 4 above)
Comment (manugoyal) [inline shell/src/test/resources/log4j.properties]: Now there's no shell tests, since they're all in integration tests now. This was the only file left over.

**Comment by apc999** [comment]
+1 LGTM

**Comment by gpang** [comment]
LGTM
